packages/waku/src/lib/middleware/rsc/ssr.ts

```diff
@@ -303,16 +304,20 @@ export const renderHtml = async <Context>(
             get(_target, name: string) {
               const file = filePath.slice(config.basePath.length);
               if (command === 'dev') {
-                const filePath = file.startsWith('@fs/')
-                  ? file.slice(3)
-                  : path.join(config.rootDir, config.srcDir, file);
+                const filePath = normalizePath(
+                  file.startsWith('@fs/')
+                    ? file.slice('@fs/'.length)
```

Comment:
Is it intentional? `3 !== '@fs/'.length`

Reply:
mistake I think, auto-complete by copilot 🤔